src/Badge/Model/UseCase/CreateDependentsBadge.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/*
+ * This file is part of the badge-poser package.
+ *
+ * (c) PUGX <http://pugx.github.io/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace PUGX\Badge\Model\UseCase;
+
+use PUGX\Badge\Service\TextNormalizer;
+use PUGX\Badge\Service\NormalizerInterface;
+use PUGX\Badge\Model\PackageRepositoryInterface;
```

Comment:
The namespace is changed in new version. See https://github.com/PUGX/badge-poser/blob/3dcba5bc1d3141d24dec4109369d4e8471b4bdcf/src/Badge/Model/UseCase/CreateVersionBadge.php#L11
Thanks